packages/ui-tray/src/Tray/README.md

```diff
@@ -6,27 +6,87 @@ The Tray is an actionable container that is triggered by click and does not need
 
 > Note that the `size` property only applies when the Tray is positioned at `start` or `end` and defines the width of the Tray.
 
-```js
----
-type: example
----
-const FPO = lorem.paragraph()
-class Example extends React.Component {
-  constructor (props) {
-    super(props)
-    this.state = {
-      open: false
+- ```js
+  const FPO = lorem.paragraph()
+  class Example extends React.Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        open: false
+      }
     }
-  }
 
-  hideTray = () => {
-    this.setState({
-      open: false
-    })
+    hideTray = () => {
+      this.setState({
+        open: false
+      })
+    }
+
+    renderCloseButton() {
+      return (
+        <Flex>
+          <Flex.Item shouldGrow shouldShrink>
+            <Heading>Hello</Heading>
+          </Flex.Item>
+          <Flex.Item>
+            <CloseButton
+              placement="end"
+              offset="small"
+              screenReaderLabel="Close"
+              onClick={this.hideTray}
+            />
+          </Flex.Item>
+        </Flex>
+      )
+    }
+
+    render() {
+      return (
+        <div style={{ padding: '0 0 16rem 0', margin: '0 auto' }}>
+          <Button
+            onClick={() => {
+              this.setState({ open: true })
+            }}
+            ref={(c) => (this._showButton = c)}
```

Comment:
this doesnt seem to do anything, you can delete it

Reply:
Done